Source/WebKit/UIProcess/API/Cocoa/_WKAttachment.h

```diff
@@ -45,6 +45,7 @@ WK_CLASS_AVAILABLE(macos(10.14), ios(12.0))
 @property (nonatomic, readonly, nullable) NSString *filePath;
 @property (nonatomic, readonly, nullable) NSData *data;
 @property (nonatomic, readonly, nullable) NSFileWrapper *fileWrapper;
+@property (nonatomic, readonly) BOOL preserveFidelity;
```

Comment:
I was expecting availability declarations on this

Reply:
Added.